README.md

```diff
@@ -1,12 +1,9 @@
 # Awesome Appwrite [![Awesome](images/badge.svg)](https://github.com/sindresorhus/awesome)
 
-![logo](images/logo.png "Logo")
-
 ## Table of Contents:
```

Comment:
In the future, we probably don't need to include this Table of Contents since GitHub has it already.